packages/rpc-api/src/simulateTransaction.ts

```diff
@@ -127,7 +127,7 @@ export interface SimulateTransactionApi extends RpcApiMethods {
     simulateTransaction(
         base58EncodedWireTransaction: Base58EncodedBytes,
         config?: SigVerifyAndReplaceRecentBlockhashConfig & SimulateTransactionConfigBase,
-    ): SimulateTransactionApiResponseBase & { accounts: null };
+    ): SimulateTransactionApiResponseBase & SolanaRpcResponse<{ readonly accounts: null }>;
```

Comment:
I think this points out a broader issue.

In lots of places, we're including the `RpcResponse` wrapper in the "base response", but in others, we do something like:

```
RpcResponse<Base & { ... }>
```

I think it should always be the latter. I'll make a note to comb for this in my pass over RPC stuff.